SourceX/miniwin/misc.cpp

```diff
@@ -19,6 +19,12 @@
 #ifndef SDL1_VIDEO_MODE_FLAGS
 #define SDL1_VIDEO_MODE_FLAGS SDL_SWSURFACE
 #endif
+#ifndef SDL1_VIDEO_MODE_WIDTH
+#define SDL1_VIDEO_MODE_WIDTH nWidth

```

Comment:
Maybe a bit of a hack considering the default can't be used in places where `nWidth` isn't defined (a local of `SpawnWindow`). Maybe instead we should `#ifdef`

Reply:
I agree but it's used in 2 places so ifdefs would get a bit verbose.
I think it's fine as this is confined to this `cpp` file.